extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -695,14 +693,14 @@ private function doMultiRemoveTags( ) {
 
 		/* since we /hopefully/ removed some tags from wikis,
 			force the search results for this pageload to be empty. */
-		$this->mTagWikiIds = array();
-		#print "(forcing mTagWikiIds to null at ".gmdate('r').")";
+		$this->mTagWikiIds = [];
+		# print "(forcing mTagWikiIds to null at ".gmdate('r').")";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)